test/functional/puppetclass_lookup_keys_controller_test.rb

```diff
@@ -14,7 +14,7 @@ class PuppetclassLookupKeysControllerTest < ActionController::TestCase
 
   test "should update lookup_keys" do
     lkey = FactoryGirl.create(:puppetclass_lookup_key, :as_smart_class_param, :puppetclass => puppetclasses(:one), :override => true, :default_value => 'test')
-    put :update, {:id => lkey.to_param, :lookup_key => { :description => "test that" }}, set_session_user
+    put :update, {:id => lkey.to_param, :puppetclass_lookup_key => { :description => "test that" }}, set_session_user
```

Comment:
Interesting, it looks like this test didn't do anything until this change, since `params[resource_name]` was nil before. Can you add a line below to confirm the `lookup_key` was updated?